include/bsppp/BSPPack.h

```diff
@@ -2,10 +2,12 @@
 
 #include <array>
 
-#include <bsppp/bsppp.h>
+#include <vpkpp/format/ZIP.h>
 
-#include "ZIP.h"
+#include "bsppp.h"
 
+// this extends off of vpkpp I think it's fine to keep it under the vpkpp  - Eden
+// also BSP already exists under the bsppp namespace
```

Comment:
honestly i think if youre going to have bsppp depend on vpkpp it wouldnt be a bad idea to merge those two classes

Reply:
oh yeah fair it might be worth looking into.

Reply:
should be done?